app/services/email/subscription.rb

```diff
@@ -0,0 +1,23 @@
+class Email::Subscription
+  attr_reader :user, :list_id, :encrypted_email
+
+  def initialize(user)
+    @user = user
+    @list_id = ENV['MAILCHIMP_NEWSLETTER_LIST_ID']
+    @encrypted_email = encrypt_email_address
```

Comment:
You don't need this; the `encrypt_email_address` method is suffices. Don't need to make it an instance variable.